custom/ewsghana/reports/specific_reports/reporting_rates.py

```diff
@@ -92,6 +87,16 @@ class ReportingDetails(ReportingRatesData):
     title = _('Reporting Details')
 
     @property
+    def title(self):
```

Comment:
does this clash with the title defined right above it? could make sense to also return `_('Reporting Details')` instead of the empty string if the `if` conditions fail